baracoda/barcodes.py

```diff
@@ -17,6 +17,7 @@
 @bp.post("/barcodes_group/<prefix>/new")  # type: ignore
 def get_new_barcode_group(prefix: str) -> Tuple[Any, int]:
     try:
+        logger.debug(f"Creating a barcode group for '{ prefix }'")
```

Comment:
There seem to be quite a few logger debugs. Are these necessary? If we start using this as oour primary source for creating barcodes will it just create a lot of noise?

Reply:
We have a setting to specify the level of logs that will be enable. I've seen in code it was set up to DEBUG, but I'll change it to ERROR (only log when there is an error or something more critical). This way it won't log this lines unless we start the application in DEBUG mode, so they can be useful to debug in local if we need. I'll do the change in config.

Reply:
Changed log level